docs/user_guides/fs/feature_view/batch-data.md

```diff
@@ -32,7 +32,7 @@ To retrieve the primary key(s) and/or event time when retrieving batch data for
     ) # return a dataframe with primary keys and event time
     ```
 !!! note
-    If the event time columns have the same name across all the feature groups included in the feature view, then only the event time of the label feature group (left most feature group in the query) will be returned. If they have different names, then all of them will be returned. The Join prefix does not have any influence on this behaviour.
+    All primary and event time columns of all the feature groups included in the feature view will be returned. If they have the same names across feature groups and the join prefix was not provided then reading operation will fail with ambiguous column exception.
```

Comment:
add a proposed solution?

Reply:
may be I wrote in negative context `if the join prefix was not provided`.